particle-accelerator/.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  extends: ['erb', 'eslint-config-brave'],
```

Comment:
Hmmm, I'm curious how well this works. What happens when there are conflicting rules between the `erb` set and the ones in `eslint-config-brave`?

Reply:
I removed it from the dependencies, I thought that it would provide the Airbnb side of the linter, but I think that the brave linter can just stand on its own.